### DISCUSSION TITLE
404 errors when trying to GET asset files during local testing

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#4093] [Q&A]
When trying to view changes that I've made to a documentation site locally on Windows 11, I'm getting 404 errors when trying to GET assets like CSS files and JavaScript files. 

![image](https://user-images.githubusercontent.com/63835446/177249569-0b00a19e-cb54-418f-af60-cd2616079525.png)

These files don't exist in my repository, and the filenames seem to indicate that these are temporary files (eg: `main.4a0965b7.min.css` and `bundle.f758a944.min.js`). I also note that the production site is hosted on GitHub and everything works fine there, as well as for a colleague who is running things on Ubuntu 20.04 (possibly 22.04).

Is there a setting that I am missing, or something that would explain these 404 errors?

**Comment by squidfunk** [comment]
I'm not sure where this could be coming from. Those are not temporary files, but the actual application files with content hashes to ensure that caches are busted. Please explain the exact steps you're doing that lead to the issue – I'm convinced that it can't just be `mkdocs serve`.
